test/e2e/specs/default-config/recordedit/multi-form-input.spec.js

```diff
@@ -0,0 +1,739 @@
+const chaisePage = require('../../../utils/chaise.page.js');
+const recordEditPage = chaisePage.recordEditPage;
+const recordEditHelpers = require('../../../utils/recordedit-helpers.js');
+
+
+const testParams = {
+  schema_table: 'multi-form-input:main',
+  max_input_rows: 200,
+  apply_tests: {
+    not_ci: true,
+    types: [
+      {
```

Comment:
So the following is how we're doing the apply tests:

1. Open the recordedit page and clone more forms.
2. Select all the forms and apply a value to all of them.
3. Deselect some forms and see that we can apply to some.
4. Deselect more forms and see that we can clear some.
5. Manually change some values in some forms.

Therefore,
- Instead of hardcoding that you're cloning 4 more forms. Just add a variable called `number_of_forms` where it's value would be "5".
- `apply_to_all_value` makes sense.
- For the "apply to some" you need two variables. One is the value and the other one would be forms that you are deselecting.
- For the "clear some" you need the forms that you're deselecting.

So the whole thing would be

```
apply_tests: {
  number_of_forms: number,
  types: [
    {
      type: string,
      column_name: string,
      apply_to_all_value: string,
      apply_to_some: {
       value: string,
       deselected_forms: number[]
      },
      clear_some_deselected_forms: number[]
      manual_test: {
        value: string,
        form: number
      }
    }
  ],
  submission: ...
}
```

We have 5 forms, 
1. apply all will change all. 
2. with apply some deselect 2, which means changing 3 forms.
3. for clear some deselect another 2, which means clearing value for 1 form.
4. Change the value of one of the forms.

It's ok if you apply some/clear some the same forms. But for the manual value change please use different forms for each type/column.